hack/run_test

```diff
@@ -372,15 +376,19 @@ function run_master_restart_test() {
   local PASS=master
 
   echo "Testing failed master restart"
-  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
   local cid_suffix="mrestart"
+  local network=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
> How that is possibe that the tests are run in parallel now in RHEL CI?

Well I would think it shouldn't be a problem to run make test VERSION=9.x for each of the versions in parallel even with the current state of the Makefile. But I don't have access to the RHEL CI so I can't check if that is really what is going on.